uitools/examples/qml_quick/src/demos/AuthenticationViewDemoForm.qml

```diff
@@ -16,6 +16,7 @@ import Esri.ArcGISRuntime 100.13
 import Esri.ArcGISRuntime.Toolkit 100.13
 
 DemoPage {
+    //! [Set up Authentication View QML]
```

Comment:
❌   I agree with Lucas we shouldn't expose any of the `mapViewContents: Component {` code. It confuses the story in how to set up an AuthenticationView.

Would it be fair to guess you've done this because you want to expose the URL? I don't think that is necessary for the url to be in the example docs. Perhaps a good middle ground is to rename `url` to something more descriptive like `exampleUrl`, then the snippet can be displayed like this:

```qml
//! [Set up Authentication View QML
MapView {
    Map {
        initUrl: exampleUrl
    }
    AuthenticationView {
        anchors.centerIn: parent
    }
}
//! [Set up Authentication View QML]
```

Reply:
Yes I wanted to expose the url, but I like the idea to imply that they will need a url by using `exampleUrl`